frontend/packages/dev-console/src/components/gitops/details/GitOpsDetailsPageHeading.tsx

```diff
@@ -21,11 +21,11 @@ const GitOpsDetailsPageHeading: React.FC<GitOpsDetailsPageHeadingProps> = ({
   const { t } = useTranslation();
   const breadcrumbs = [
     {
-      name: t('devconsole~Application Stages'),
-      path: '/applicationstages',
+      name: t('devconsole~Environments'),
+      path: '/environments',
     },
     {
-      name: t('devconsole~Application Details'),
+      name: t('devconsole~Environments'),
```

Comment:
Should this be devconsole~Application Environments?  If so, this _should_ fix the test.

Reply:
Yes this should be `devconsole~Application Environments`. @reginapizza please fix this.